app/src/main/index.ts

```diff
@@ -10,8 +10,12 @@ openDatabase();
 
 function createWindow(): void {
   const mainWindow = new BrowserWindow({
-    width: 900,
-    height: 670,
+    width: is.dev ? 1200 : 900,
+    height: 700,
+    minWidth: 900,
+    minHeight: 700,
+    title: "SecureDrop Client",
```

Comment:
I'd suggest "SecureDrop App" throughout, that's what we've been using for the milestones, and it distinguishes it usefully from the Client - besides, Client was always the more technical term.